angular.json

```diff
@@ -779,6 +779,111 @@
                     "defaultConfiguration": "development"
                 }
             }
+        },
```

Comment:
![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/2b2c585b-29d9-4dfa-91e7-b7342fe036bb)

Maybe we should re-order:

* Name + Display Name
* Type
* Description

at least the name should be on top.

Reply:
6cd9b2f4b01354d513112c01af3f58fc468b14d6

Reply:
![grafik](https://github.com/geo-engine/geoengine-ui/assets/4947061/9a698987-4614-43bb-9cae-2e6e9b4d6f85)

Now, that you've put things side-by-side you should add a margin in-between.

(And… we could think of allowing wrapping into a new line if the screen gets small.)

Reply:
I added gaps in the rows: 39c9c59e86e4906adeee802703a940d90088a43b

Does making the stuff responsive have to be part of the very first version? I think there is more business value elsewehere.